app/src/operations/mail/index.ts

```diff
@@ -5,7 +5,7 @@ export default defineOperationApp({
 	icon: 'mail',
 	name: '$t:operations.mail.name',
 	description: '$t:operations.mail.description',
-	overview: ({ subject, to, body }) => [
+	overview: ({ subject, to, type, body }) => [
```

Comment:
> But the current issue is, the dropdown is not saving the default option value which is Markdown, but that's not broke anything, in fact, everything still works normally

This is technically working as intended  as v-form doesn't "save" (or stage) default values, but yeah it can be odd in that it doesn't show the "default value" in operation panel overview in this case.

On second thought, perhaps we can just remove this label from overview? Since it's not as important, and other flows operations also only shows certain options, not all.

(an alternative fix if `type` ended up being preferred to be shown, may be doing `text: type || 'input-rich-text-md'`)